docs/ensembl-help/using-ensembl/ensembl-apps/entity-viewer/transcripts.md

```diff
@@ -14,12 +14,55 @@ status: draft
 
 # What’s in the transcripts view?
 
-Transcript view lists all transcripts of the focus gene. You can find the graphical deception of the transcript models with exons illustrated as boxes connected by lines representing introns. Coding exons are coloured, while non-coding exons are empty.
- 
-Transcript models are positioned relative to the gene start and end, so that the yellow grid matches the coordinates in the gene image above. Click on the transcript image or its corresponding Ensembl transcript stable IDs on the right to show/hide information about this transcript. Where avialbale, the first transcript on the list is MANE Select and is expanded by default.
- 
-The expanded view includes transcript biotype, genomic coordinates, exon count, combined exon length in bp and amino acid length of its translation for coding transcripts. Ensembl protein stable ID of the corresponding translation links out to the Gene function view. Transcript flags, such as APPRIS, GENCODE basic, TSL and RefSeq match for MANE Select transcripts can be found under More information. There is also an option to view this transcript in the Genome browser. 
- 
-Expanded transcript view allows quick sequence Download for both the gene and the focus transcript. Click Download to reveal customisation options, select your preference by ticking the boxes and hit the green Download button to export sequence data in FASTA format. You can select multiple options at once.
- 
-You can manipulate the transcript list by clicking Filter & sort on the left. The expanded menu offers options to filter the list by transcript biotype and scores (multiple choice) and to sort by exon length and count. Changes to the transcript list in this tab will be applied to all subsequent views.
+Information about all the transcripts of a gene of interest is displayed in the Transcript panel. 
+
+Each transcript is represented graphically as a white on a black background and labelled with its Ensembl transcript stable ID.
+
+Transcripts are positioned relative to the gene start and end, so that the yellow grid matches the coordinates in the gene image above. 
+
+
+## Introns and exons
+
+Exons are depicted as white boxes and introns as the line connecting the exons. Coding exons are coloured, while non-coding exons are empty.
+
+The Transcript panel view is set by default in the Entity viewer.
```

Comment:
Do you mean 'open for MANE Select or Ensembl canonical by default' ?

Reply:
done